index.html

```diff
@@ -204,6 +204,7 @@ <h2 id="footer-heading">CONTACT US</h2>
      </form>
```

Comment:
- [ ] [OPTIONAL] PLEASE try to add some padding to your inputs  in order to match the Figma design 
- yours
- ![image](https://user-images.githubusercontent.com/61790587/172275336-ff6562d6-8022-4676-9ed1-7c0af772cfc4.png)